_posts/2017-03-15-Performance-of-LSTM-Using-CuDNN-v5.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Performance of LSTM with or without cuDNN(v5) in Chainer
+layout: post
+categories: General
+---
+
+We compare the performance of LSTM with or without cuDNN in Chainer.
+The NVIDIA CUDA® Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.
+cuDNN provides highly tuned implementations for standard routines such as LSTM, CNN.
+
+In this article, we compare the performance of LSTM with or without cuDNN.
+An LSTM (NStepLSTM) implementation is included in Chainer, which can be found here.
```

Comment:
From this description, I think it is not clear what the difference is between the "LSTM without cuDNN" and the "LSTM with cuDNN". Is the NStepLSTM implementation configurable to run with or without cuDNN? Or should the reader assume that NStepLSTM does not use cuDNN but there is some other code that is optimized to use cuDNN that was used for the benchmarking?

Maybe it would help the reader to understand if you could describe the differences between the two LSTM implementations (with or without cuDNN) that were used.